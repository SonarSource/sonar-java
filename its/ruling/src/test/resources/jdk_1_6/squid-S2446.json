{
'jdk_1_6:java/util/Timer.java':[
84,
367,
389,
],
'jdk_1_6:java/util/prefs/AbstractPreferences.java':[
1534,
1548,
1562,
],
}
