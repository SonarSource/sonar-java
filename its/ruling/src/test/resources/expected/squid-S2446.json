{
'project:com/sun/corba/se/impl/corba/AsynchInvoke.java':[
82,
],
'project:com/sun/corba/se/impl/encoding/BufferManagerReadStream.java':[
69,
91,
],
'project:com/sun/corba/se/impl/orb/ORBImpl.java':[
793,
],
'project:com/sun/corba/se/impl/orbutil/concurrent/CondVar.java':[
239,
302,
334,
],
'project:com/sun/corba/se/impl/orbutil/concurrent/DebugMutex.java':[
159,
172,
206,
],
'project:com/sun/corba/se/impl/orbutil/concurrent/Mutex.java':[
147,
155,
186,
],
'project:com/sun/corba/se/impl/orbutil/concurrent/ReentrantMutex.java':[
188,
231,
263,
289,
338,
],
'project:com/sun/corba/se/impl/orbutil/threadpool/ThreadPoolImpl.java':[
224,
],
'project:com/sun/corba/se/impl/transport/CorbaResponseWaitingRoomImpl.java':[
245,
289,
],
'project:com/sun/corba/se/impl/transport/SocketOrChannelConnectionImpl.java':[
1010,
],
'project:com/sun/jmx/remote/internal/ServerCommunicatorAdmin.java':[
96,
125,
],
'project:com/sun/jmx/snmp/tasks/ThreadService.java':[
87,
175,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/CoroutineManager.java':[
273,
317,
344,
],
'project:java/util/Timer.java':[
114,
410,
432,
],
'project:java/util/prefs/AbstractPreferences.java':[
1549,
1563,
1577,
],
'project:javax/swing/text/AbstractDocument.java':[
1423,
],
}
